experimental/language-generation/README.md

```diff
@@ -95,10 +95,24 @@ Quite often your bot might target more than one spoken/ display language. To do
 The current library does not include any capabilities for grammar check or correction.
 
 ## Packages
+Latest preview packages are available here
+- C# -> [NuGet][14]
+- JS -> [npm][15]
 
-Packages for C# are available under the [BotBuilder MyGet feed][12]
+Nightly packages for C# are available here
+- C# -> [BotBuilder MyGet feed][12]
+- JS -> [BotBuilder MyGet feed][13]
 
 ## Change Log
+### 4.7 PREVIEW
+- \[**BREAKING CHANGES**\]:
+    - Old way to refer to a template via `[TemplateName]` notation is deprecated in favor of `@{TemplateName()}` notation.
```

Comment:
Perhaps consider a link to example syntax a bit like we worked through on mail the other week?